exp/runtime/api/v1beta1/extension_types.go

```diff
@@ -0,0 +1,197 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

Comment:
Let's make sure we follow the convention:

lowerCase(Kind)_types.go = `extensionconfig_types.go` (same for other files)